### PR TITLE
Cypress/E2E: Fix MM-T1462 accessibility for incoming message

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
- Fix MM-T1462 accessibility for incoming message
- Add fix to MM-T3161 channel sidebar

#### Ticket Link
none, failing on daily Cypress run.

Screen Shot 2020-10-01 at 1 49 47 AM

Screen Shot 2020-10-01 at 1 54 24 AM

### CODE REVIEW

**Review by josephbaylon — COMMENTED** [review]
The original issues were fixed but for `custom_categories_spec` I'm getting this error consistently locally,
Screen Shot 2020-09-30 at 11 28 20 AM

File: e2e/cypress/integration/channel_sidebar/custom_categories_spec.js [hunk 1]
```diff
@@ -129,13 +125,20 @@ function createCategoryFromSidebarMenu() {
...
+    // # Click on the sidebar menu dropdown
+    cy.findByLabelText('Add Channel Dropdown').click();
 
     // # Click on create category link
-    cy.get('#createCategory').click();
+    cy.findByText('Create New Category').should('be.visible').click();
+
+    // # Verify that Create Category modal has shown up.
+    // # Wait for a while until the modal has fully loaded, especially during first-time access.
+    cy.get('#editCategoryModal').should('be.visible').wait(TIMEOUTS.HALF_SEC).within(() => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/channel_sidebar/custom_categories_spec.js:136]: @josephbaylon Thanks for testing. Looks like the issue is here, especially during first-time access to the modal. Please try again and see if it works for you.

Also, while here, I've fixed others by:
- using `cy.findByLabelText` when targeting with `aria-label`,
- capitalize comments for consistency
- fixed some typo errors

**Review by josephbaylon — APPROVED** [review]
LGTM. Tests passed locally.

**Review by srkgupta — APPROVED** [review]
Tested and passed. Changes LGTM

**Pull request merged**